src/ground-truth.rkt

```diff
@@ -53,17 +53,19 @@
         'unsamplable)
        y))))
 
-(define (eval-prog-real prog ctx)
```

Comment:
Did you fix up the other caller of `eval-prog-real`? I believe there is a call in `bsearch.rkt`.

Reply:
I think I got all of them. CI wouldn't pass or compile unless I did right?